content/blog/2021/outreach-partners.md

```diff
@@ -0,0 +1,40 @@
+Title: We are looking for outreach partners for PyCon India 2021
+Date: 2021-04-06 23:00
+Author: Nikhil Maan (@sc0rpi0n101)
```

Comment:
Can we start removing this and change this to PyCon India Content Team? We had discussed this in the past. A new year. so it would be good to revert to the older tradition.